### PR TITLE
Update v-model.md naming convention

### PR DESCRIPTION
**Pull request by leochoo**
## Description of Problem
camelCase is being used in the example, not following the convention.

## Proposed Solution
kebab-case should be used when event is being listened to inside parent's template.

## Additional Information
#2503

### CODE REVIEW

**Review by AloisSeckar — CHANGES_REQUESTED** [review]
while we are suggesting the changes, the prop itself should also be hypethenated here:

image

File: src/guide/components/v-model.md
Comment (AloisSeckar) [inline src/guide/components/v-model.md:24]: should be :model-value="searchText"

**Comment by leochoo** [comment]
@AloisSeckar 
Should we also change this part?
```
1. Bind the `value` attribute of a native `<input>` element to the `modelValue` prop
2. When a native `input` event is triggered, emit an `update:modelValue` custom event with the new value
```

Hesitant because they explain the use of camelCase in [Component Events](https://vuejs.org/guide/components/events.html#emitting-and-listening-to-events)

> Like components and props, event names provide an automatic case transformation. Notice we emitted a camelCase event, but can listen for it using a kebab-cased listener in the parent. As with [props casing](https://vuejs.org/guide/components/props#prop-name-casing), we recommend using kebab-cased event listeners in templates.

**Comment by AloisSeckar** [comment]
> @AloisSeckar Should we also change this part?

No, this is ok. The attribute VALUES inside child component must comply with the props and emits definition, which is:

```
<script setup>
defineProps(['modelValue'])
defineEmits(['update:modelValue'])
</script>
```

But the corresponding attribute NAMES in parent should be kebab-cased to avoid possible clashes.

Because from the JavaScript point of view "modelValue" and "modelvalue" are two distinct things, while html attributes names are [case-insensitive by definition](https://www.w3.org/TR/2010/WD-html-markup-20101019/documents.html#case-insensitivity).

**Comment by leochoo** [comment]
Thanks for confirming! I think we can merge this after your approval now!

**Review by NataliaTepluhina — APPROVED** [review]

**Pull request merged**
